pkg/diskboot/device.go

```diff
@@ -8,8 +8,7 @@ import (
 	"fmt"
 	"io/ioutil"
 	"path/filepath"
-	"strings"
-
+	"github.com/u-root/u-root/pkg/storage"
```

Comment:
oops, I made a mistake. The ...storage path needs to be in the section below. You can put it after ...mount